client/src/js/services/SubsidiesService.js

```diff
@@ -0,0 +1,87 @@
+angular.module('bhima.services')
+  .service('SubsidiesService', SubsidiesService);
```

Comment:
A better name would be `SubsidyService`.  `Subsidie` isn't a valid English spelling :).